draft-ietf-acme-acme.md

```diff
@@ -2597,6 +2618,33 @@ within their trusted network and use these resolvers both for both CAA record
 lookups and all record lookups in furtherance of a challenge scheme (A, AAAA,
 TXT, etc.).
 
+## External Accounts
```

Comment:
The discussion in Seoul also mentioned the use of ACME account keys to bind existing accounts together.  That is, where the CA has forms that allow a user to enter details of their ACME accounts. 

I think that we want advice on how to construct a token that proves to a CA that the account key is controlled by the current user.  This is a new use of the account key outside the ACME protocol, which is potentially risky if the user can be convinced to sign an arbitrary challenge.  (Cross domain usage, etc...)  Advice on construction is probably useful.  Standardizing the form would be better.